components/playlist/browser/resources/components/app.v1.tsx

```diff
@@ -48,6 +50,7 @@ export default function App() {
         const playlistId = match?.params.playlistId
         return (
           <AppContainer isPlaylistPlayerPage={!!playlistId}>
+            <AlertCenter />
```

Comment:
@AlanBreck mentioned in this thread https://bravesoftware.slack.com/archives/C03A0MWJXP0/p1702861612151929 that you might be able to position it using `position: relative` or `position: static`

Life might be easier without using the AlertCenter though, not sure:

```tsx
    {failedToPlay && <Alert>Oh no! We failed</Alert>}
```

Reply:
It seems like the current Nala version doesn't include the change for the variable. Let's catch up on this later.